File: Demos/Servlet/src/main/java/com/groupdocs/ui/signature/model/web/SignatureFileDescriptionEntity.java
package com.groupdocs.ui.signature.model.web;

import com.groupdocs.ui.model.response.FileDescriptionEntity;

/**
 * SignatureFileDescriptionEntity
 *
 * @author Aspose Pty Ltd
 */
public class SignatureFileDescriptionEntity extends FileDescriptionEntity {
    private String image;

    /**
     * Get incoded image Base64 string
     * @return image
     */
    public String getImage() {
        return image;
    }

    /**
     * Set incoded image Base64 string
     */
    public void setImage(String image) {
        this.image = image;
    }
}

File: Demos/Ktor/src/main/kotlin/com/groupdocs/ui/signature/ktor/model/SignatureFileDescriptionEntity.kt
package com.groupdocs.ui.signature.ktor.model

open class SignatureFileDescriptionEntity(
     override val guid: String? = null,
     override val name: String? = null,
     override val docType: String? = null,
     override val directory: Boolean? = null,
     override val size: Long? = null,

) : FileDescriptionEntity()